docs/core/development/policy/compatibility-policy.rst

```diff
@@ -180,6 +180,13 @@ Even though Python objects can be pickled and unpickled without explicit support
 Because of this, no application may depend on any object defined by Twisted to provide pickle compatibility between any release unless the object explicitly documents this as a feature it has.
 
 
+Representations
+^^^^^^^^^^^^^^^
+
+The printable representaions of objects, as returned by ``repr(object)`` and defined by ``def __repr__(self):`` are for debugging and informational purposes.
+Because of this, no application may depend on any object defined by Twisted to provide repr compatability between any release.
```

Comment:
I also considered a note about how applications should also not rely on `eval(repr(object)) == object` compatability

Reply:
Typos: "compatability" -> "compatibility" and "representaions" -> "representations".

Also, perhaps "applications must not depend" is clearer than "no application may depend".